Makefile

```diff
@@ -20,23 +18,31 @@ ci: get_tools get_vendor_deps install test_cover test_lint test
 ########################################
 ### Build/Install
 
-check-ledger:
+# Add ledger support - Makefile variable changes are out of a target's scope. There is no point in putting this into a target.
```

Comment:
Is this a TODO? Im confused?

Reply:
Nope, it's a comment. We can remove it. The point is that checking the ledger support is not a target, it's a Makefile variable set - which is not a scope of a target.

Reply:
Ok, then I don't think we need the comment then 👍